orchestra/static/orchestra/team-info/team-info-card.directive.es6.js

```diff
@@ -8,35 +8,58 @@ export default function teamInfoCard (orchestraApi) {
     template,
     restrict: 'E',
     scope: {
-      projectId: '='
+      taskAssignment: '='
```

Comment:
refactor: don't pass project ID---pass task assignment, which contains a project pointer and other facts that we now need